etc/DependencyInstaller.sh

```diff
@@ -698,6 +750,9 @@ EOF
             fi
             _installOrTools "ubuntu" "${version}" "amd64"
         fi
+        if [[ ${CI} == "yes" ]]; then
+            _installCI
+        fi
```

Comment:
A warning message would be nice if the user tries to install CI deps in an unsupported OS